src/http/request.ts

```diff
@@ -174,6 +174,17 @@ export class Request extends ServerRequest {
     return null;
   }
 
+  /**
+   * Gets a record whose keys are the request's url query params specified by inputs
+   * and whose values are the corresponding values of the query params.
+   * 
+   * @param inputs url query params
+   * @returns null if no inputs or record containing key-value pairs
```

Comment:
Just need to remove the `@param` line and adjust the `@returns` description to say `Key value pairs of the query param and it's value. Empty object if no query params`,  just to be inline with TSDoc

Reply:
This is fine.